addons/account/static/src/components/account_payment_field/account_payment_field.js

```diff
@@ -14,6 +14,24 @@ import { Component } from "@odoo/owl";
 class AccountPaymentPopOver extends Component {
     static props = { "*": { optional: true } };
     static template = "account.AccountPaymentPopOver";
+
```

Comment:
```suggestion
        setup() {
          this.memoContent = useRef("memoContent");
          this.memoTooltip = useRef("memoTooltip");
      }
```
I believe you could use the `useRef` owl function which is specifically designed for that. Also, as you use memoTooltip both in `onMemoHover` and `onMemoLeave`, it's better to create the variable in the `setup` function. However, with this way of working, you'll need to write things like this `this.memoContent.el.scrollHeight` for example. But at least it is the proper way of doing thing I believe. Also, you'll need to use `t-ref="memoContent"` and `t-ref="memoTooltip"` in the template. Here is the [doc](https://github.com/odoo/owl/blob/master/doc/reference/refs.md)

Reply:
Yeah, indeed it will be cleaner and consistent with other similar usages in Odoo.